addons/web/static/src/less/list_view.less

```diff
@@ -117,6 +117,18 @@
         border-top: none;  // Override bootstrap for grouped list views
     }
 
+    tbody {
+        tr:focus-within {
+            background-color: @odoo-form-lightsecondary;
+        }
+        tr{
+            .focussed {
```

Comment:
http://www.future-perfect.co.uk/grammar-tip/is-it-focussed-or-focused/ (focused is highly preferred)